src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -181,3 +181,31 @@ export const recordReleaseWarning = (
     <span>Limiting consent means that your doctor can only share records that are directly related to your condition. This could add to the time it takes to get your private medical records.</span>
   </div>
 );
+
+export const documentDescription = () => {
+  return (
+    <div>
+      <p>File upload guidelines:</p>
+      <ul>
+        <li>File types you can upload: .pdf, .jpeg, or .png</li>
+        <li>Maximum file size: 50 MB</li>
+      </ul>
+      <p><em>Large files can be more difficult to upload with a slow Internet connection</em></p>
+    </div>
+  );
+};
+
+export const additionalDocumentDescription = () => {
+  return (
+    <div>
+      <p>If you have other evidence, like lay or buddy statements, that you would like to submit, you can upload them here.</p>
+      <p>File upload guidelines:</p>
+      <ul>
+        <li>File types you can upload: .pdf, .jpeg, or .png</li>
+        <li>Maximum file size: 50 MB</li>
+      </ul>
+      <p><em>Large files can be more difficult to upload with a slow Internet connection</em></p>
```

Comment:
Should this end with a `.`?

Reply:
Good question for @peggygannon. Peggy should this sentence include a period at the end?
*Large files can be more difficult to upload with a slow Internet connection*
![screen shot 2018-03-21 at 4 53 52 pm](https://user-images.githubusercontent.com/16051172/37743683-7a1f66b6-2d28-11e8-8a15-03c07aa61b88.png)